packages/shared/src/variants/s_fractional.ts

```diff
@@ -308,6 +313,32 @@ export class SFractional extends AbstractGame<
       secondary_colors: ["#0000FF", "#FF0000"],
     };
   }
+
+  static uiTransform(
+    config: SFractionalConfig,
+    gamestate: SFractionalState,
+  ): {
+    config: SFractionalConfig & DefaultBoardConfig;
+    gamestate: DefaultBoardState;
+  } {
+    const boardShape = isGridBadukConfig(config) ? "2d" : "flatten-2d-to-1d";
+
+    const stoneTransform = (colors: PlacementColors): MulticolorStone => {
+      return { colors: colors };
+    };
+    const scoreTransform = (color: string): string[] | null =>
+      color === "" ? null : [color];
```

Comment:
Null checking not necessary here because `SFractionalState.score_board` is `string[][]` if it exists.

```suggestion
    const scoreTransform = (color: string): string[] => [color];
```

Reply:
Oh you're totally right. I misread it as saying color could be null.